src/core/js/logging.js

```diff
@@ -0,0 +1,78 @@
+define(['core/js/adapt'], function(Adapt) {
```

Comment:
The 4 for indentation conflicts with: https://github.com/adaptlearning/documentation/blob/master/01_cross_workstream/style_guide.md which is linked in https://github.com/adaptlearning/adapt_framework/wiki/Contributing-code